modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +238,141 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
++
+This update fixes the issue. It passes the namespace-based configuration to the job so that all the commands run in one container in a loop. (link:https://issues.redhat.com/browse/SRVKP-2160[SRVKP-2160])
+// Pradeep Kumar
+
+* In {tekton-chains} you must define a secret called `signing-secrets` to hold the key used for signing tasks and images. However, before this update, updating the {pipelines-title} Operator reset or overwrote this secret and the key was lost. This update fixes the issue. Now, if the secret is configured after installing {tekton-chains} through the Operator, the secret persists, and it is not overwritten by upgrades.
+// https://issues.redhat.com/browse/SRVKP-2304
+// Piyush Garg
+
+* Before this update, all S2I build tasks failed with an error similar to the following message:
++
+----
+Error: error writing "0 0 4294967295\n" to /proc/22/uid_map: write /proc/22/uid_map: operation not permitted
+time="2022-03-04T09:47:57Z" level=error msg="error writing \"0 0 4294967295\\n\" to /proc/22/uid_map: write /proc/22/uid_map: operation not permitted"
+time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
+----
++
+With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah cluster tasks. As a result, the S2I build tasks run successfully.
++
+// (borrowed from 1.7.2 release notes) https://issues.redhat.com/browse/SRVKP-2091
+// Piyush Garg
+
+* Before this update, installing the {product title} Operator took longer than expected. This update optimizes some settings to make installing the Operator go faster.
+// https://issues.redhat.com/browse/SRVKP-2369
+
+* With this update, Buildah and S2I cluster tasks have fewer steps than in previous versions. Some steps have been combined into a single step so they work better with `ResourceQuota` and `LimitRange` objects and do not require more resources than necessary.
+// https://issues.redhat.com/browse/SRVKP-2333
+// Rupali Behera
+
+* This update upgrades the Buildah, `tkn` CLI tool, and `skopeo` CLI tool versions in cluster tasks.
+// https://issues.redhat.com/browse/SRVKP-2214
+// Vincent Demeester @vdemeester
+
+*  Before this update, the Operator failed while creating RBAC resources if any of the namespaces were in a "Terminating" state. With this update, the Operator ignores namespaces in a "Terminating" state and creates the RBAC resources.
+// https://issues.redhat.com/browse/SRVKP-2248
+// Shivam Mukhade
 
-// Pipelines
-// *
+* Before this update, pods for the `tkn` CLI tool and prune jobs were not being scheduled on infrastructure nodes, as expected. Instead, they were being scheduled on worker nodes or not scheduled at all. With this update, these types of pods are being scheduled on infrastructure nodes.
+// https://issues.redhat.com/browse/SRVKP-1806
+// Rupali Behera
```

Comment:
@rupalibehera  Please review and approve. Thanks.

Reply:
a minor change.